client/components/ellipsis-menu/style.scss

```diff
@@ -0,0 +1,12 @@
+.ellipsis-menu__menu hr {
```

Comment:
I think we should have a `MenuSeparator` subcomponent of PopoverMenu for this, we use it in many places (pages, themes).